src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/CreateNewListModal.tsx

```diff
@@ -48,7 +47,38 @@ const logger = createLogger(
 
 const MAX_NAME_LENGTH = 40
 
-export const CreateNewListModal: FC<CreateNewListModalProps> = ({
+const onListAdded = (store, data) => {
```

Comment:
**suggestion**: I would extract useMutation to a separate file since mutation logic grew quite a bit

Reply:
yeah, agree with Nikita